### PR TITLE
chore: sync founation-ui with docs

### PR DESCRIPTION
**Pull request by matteematt**
Thank you for contributing to the documentation.

Do the changes you have made apply to both Current and Previous versions?

No

Have you checked all new or changed links?

Yes

Is there anything else you would like us to know?

No

For reference: 

  - We have an [internal contributions guide](https://www.notion.so/genesisglobal/Contributing-new-documentation-75953fb245f246ff872789035451a0c4)
  - We have a [style guide](https://www.notion.so/genesisglobal/Documentation-style-guide-5b04ec6fe12f4262b90d192effd8059b) 

**This week's exciting excerpts from the style guide**

- We write in UK English, with Oxford English Dictionary (OED) spellings, grammar and vocabulary.  

Use the present tense wherever possible. Only use another tense where it is strictly necessary.

- Present tense (preferred whenever possible): *The script creates a new folder.*
- Future tense (avoid whenever possible): *The script will create a new folder.*

### CODE REVIEW

File: docs/04_web/12_forms/01_foundation-forms.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: 'Foundation Forms'
 sidebar_label: 'Foundation Forms'
-id: 'foundation-forms'
+id: foundation-forms
```
Comment (matteematt) [inline docs/04_web/12_forms/01_foundation-forms.md:4]: @wjhendry do you know if it is ok to remove the quotes around this? The tool seems to have done that for some reason

File: docs/04_web/12_forms/01_foundation-forms.md
Comment (matteematt) [inline docs/04_web/12_forms/01_foundation-forms.md:1]: @SzymonZur can you check if this change looks ok please? Last time I was changing the forms page I got into a bit of a muddle

**Comment by matteematt** [comment]
@wjhendry so I discussed the forms document with you on Slack, and we still need to find a long term solution there. How should we approach this for now though? Do you want me to revert the change on this PR, and then you manually ammend the three pages either in this PR or another PR to match the changes that are required from our end?

**Comment by wjhendry** [comment]
> @wjhendry so I discussed the forms document with you on Slack, and we still need to find a long term solution there. How should we approach this for now though? Do you want me to revert the change on this PR, and then you manually ammend the three pages either in this PR or another PR to match the changes that are required from our end?

Looks like you revert and I do manually. Give me 10 minutes on this.

**Comment by matteematt** [comment]
> > @wjhendry so I discussed the forms document with you on Slack, and we still need to find a long term solution there. How should we approach this for now though? Do you want me to revert the change on this PR, and then you manually ammend the three pages either in this PR or another PR to match the changes that are required from our end?
> 
> Looks like you revert and I do manually. Give me 10 minutes on this.

You're probably better asking @SzymonZur about the material changes required for the updates as he touched it last. Perhaps you're correct and there isn't anything that needs updating for now (if all the changes are reverting 3 -> 1 page). If I get confirmation of that then I can just revert the page change and leave it at that

**Comment by wjhendry** [comment]
> > > @wjhendry so I discussed the forms document with you on Slack, and we still need to find a long term solution there. How should we approach this for now though? Do you want me to revert the change on this PR, and then you manually ammend the three pages either in this PR or another PR to match the changes that are required from our end?
> > 
> > 
> > Looks like you revert and I do manually. Give me 10 minutes on this.
> 
> You're probably better asking @SzymonZur about the material changes required for the updates as he touched it last. Perhaps you're correct and there isn't anything that needs updating for now (if all the changes are reverting 3 -> 1 page). If I get confirmation of that then I can just revert the page change and leave it at that

Thumbs up for that - shall I discuss directly with @SzymonZur? Then I can let yo know

**Comment by SzymonZur** [comment]
@matteematt @wjhendry let me check if anything needs updating

**Comment by SzymonZur** [comment]
@wjhendry there have been no changes since December so everything is ok.

**Comment by matteematt** [comment]
Thanks for the confirmation, will revert the required changes later today

**Comment by matteematt** [comment]
@wjhendry @SzymonZur thanks for the input both of you, I have now reverted the forms commit so this is ready for review again

**Review by wjhendry — APPROVED** [review]

**Pull request merged**